src/Microsoft.AspNet.Razor/Chunks/Generators/TemplateBlockChunkGenerator.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Parser.SyntaxTree;
+
+namespace Microsoft.AspNet.Razor.Chunks.Generators
+{
+    public class TemplateBlockChunkGenerator : ParentChunkGenerator
```

Comment:
I debated changing the various code generators names from `Block` => `Parent` but in the end decided not to sense the more specific Generators were there to relate to their corresponding syntax tree nodes.